azure-webapp-maven-plugin/README.md

```diff
@@ -195,8 +188,21 @@ The supported values for Web App on Windows:
 `1.8.0_92` | Azul's Zulu OpenJDK 8, Update 92
 `1.8.0_102` | Azul's Zulu OpenJDK 8, Update 102
 > Note: It is recommended to ignore the minor version number so that the latest supported JVM will be used in your Web App.
+  
+
+#### `<webContainer>`
+
+The supported Value for web App on Linux
+
+Supported Value | Description
+---|---
+`tomcat 8.5` | Newest Tomcat 8.5
+`tomcat 9.0` | Tomcat 9.0
+`wildfly 14` | WildFly 14 
+  
+
+The supported Value for web App on windows
```

Comment:
Highlight the `The supported values for web App on Windows`

Reply:
OK